workflows/repeatmasking/RepeatMasking-Workflow.ga

```diff
@@ -0,0 +1,168 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
+    "license": "MIT",
+    "release": "0.1",
+    "name": "Workflow constructed from history 'RepeatMasking'",
+    "steps": {
+        "0": {
+            "annotation": "",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "",
+                    "name": "sequence.fasta"
+                }
+            ],
+            "label": "sequence.fasta",
```

Comment:
The workflow run form will automatically restrict inputs to fasta files based on the acceptable inputs this is connected to. You could drop the `.fasta` since that is redundant.